jo-rdan/Free-Mentor
# #167922093 Mentor can accept a request

#### What does this PR do?
It creates an endpoint for a **mentor** to accept a mentorship request
#### Description of Task to be completed?
- write unit tests to test if the endpoint works as expected
- add authentication and authorizations to the user who access this endpoint
#### How should this be manually tested?
- Clone this repository to your local machine
- Open the cloned folder and start the server by typing **npm install**
- in **POSTMAN** create a **POST** on the following **localhost:3000/api/v1/sessions/:id(mentor id)/accept**
- Make sure the token provided in the headers is from a mentor and the response will be a **200** HTTP status code along with the data of the accepted session request, otherwise it will respond with a relevant error message.
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#167922093](https://www.pivotaltracker.com/story/show/167922093)
